source/setup.py

```diff
@@ -234,6 +234,12 @@ def getRecursiveDataFiles(dest,source,excludes=()):
 		],
 		"packages": [
 			"NVDAObjects",
+			# As of py2exe 0.11.0.0 if the forcibly included package contains subpackages
+			# they need to  be listed explicitly (py2exe issue 113).
+			"NVDAObjects.IAccessible",
+			"NVDAObjects.JAB",
+			"NVDAObjects.UIA",
+			"NVDAObjects.window",

```

Comment:
How did you determine these - and how can we be confident that nothing is missing?

Reply:
Unfortunately this has to be done manually.
We know that if the forcibly included package has no sub-packages it is included successfully and the only one known package with sub-packages we need to include that way is `NVDAObjects`. Since no sub-packages in `NVDAObjects` has sub-packages it is sufficient to list all of them. To ensure that this procedure works I've just compared count of files inside each sub-package in the `library.zip`with amount of files in the repository.

Reply:
Thanks for the explanation, LGTM